sechub-cli/src/mercedes-benz.com/sechub/util/tar_test.go

```diff
@@ -0,0 +1,289 @@
+// SPDX-License-Identifier: MIT
+package util
+
+import (
+	"archive/tar"
+	"fmt"
+	"os"
+	"strings"
+	"testing"
+
+	sechubUtil "mercedes-benz.com/sechub/testutil"
+)
+
```

Comment:
All your tests files are created with the following Unix permissions `644` (owner+write, group+read, all+read). The folders are always created with permissions `755` (owner+full, group+list, all+list). It would be interesting to have a test, in which the permissions are set differently to see if this might cause problems on Linux, macOS and other Unix-like operating systems and if the permissions are preserved inside the tar archive.

In general, I can also think of setting the permissions of all files to `444` and all folders to `555` in the tar archive. For us it is important to read the files to be able to scan them, everything else is not essential.

Reply:
- in a tar file, all permissions and ownership are contained:
```
> tar tvf binaries-sechub_evaluation.tar
-rwxr-xr-x johndoe/users 3350469 2022-04-12 17:46 __data__/my-libs/binaries/gofmt
-rw-r--r-- johndoe/users     902 2022-03-24 11:57 __data__/my-libs/test.bin
```
So indeed, here it would be interesting how an integration test would behave with e.g. user-only read permissions and the userID differs from the user running the SecHub server.
-> Created an issue for this: https://github.com/mercedes-benz/sechub/issues/1356

- in a .zip file there are no userIDs or permissions - so here we should be safe